### ISSUE TITLE
README linux install instructions needed

### ISSUE THREAD

**Issue by snowc0de** [popcorntime/popcorntime#1417]
Hello I am writing this because the installation instructions for Linux on the README is not correct. Also, the installation for Linux in general is unclear and I don't understand why you dropped the support of DEB file after the 0.3.1 release.
Neither of the methods to install works properly. It would be a good idea to get an installable binary package unlike the AppImage.
For the rest, I link this issue with this thread of the forum:

https://discuss.popcorntime.app/t/please-update-your-readme

I noticed that this issue has already been made but I wanted to add one anyway because nobody seems to care about these issues:

#1105 #1260 

So please do not underestimate the importance of documentation in a project.

Thanks in adavnce!

**Comment by louisdoe** [comment]
I back up this. We (linux users) need a clean install procedure. Thanks !!

**Comment by Persei08** [comment]
You guys are confused here. I understand that you need an easy step by step guide or a deb file but you're making some mistake aside.

There is no Linux install instruction on the readme. The "Gettings started" instructions aren't linux specific and said:
>If you're comfortable getting up and running from a `git clone`, **this method is for you**.

So, users (devs, contributors, etc...) who clone the git repo and type the yarn commands (BTW they need to set their dev environment... installing nodejs and yarn first) should be able to build popcorntime and start it.

The `make_popcorn.sh` script (for linux and macos) is basically a script cloning the git repo and running the yarn commands.

>why you dropped the support of DEB file after the 0.3.1 release.

There is no 0.3.10 deb file [on the website](https://get.popcorntime.app/repo/build/), you probably get your [3.10 deb file from CI platform](https://ci.popcorntime.app/view/All/job/Popcorn-Time-Desktop/200/), and the [deb files are still provided there](https://ci.popcorntime.app/view/All/job/Popcorn-Time-Desktop/lastStableBuild/) (except maybe when there is an issue to generate it).

>I noticed that this issue has already been made but I wanted to add one anyway because nobody seems to care about these issues:
#1105 #1260

Every time I come here, I remove duplicate issues.. i'm bored :|

- for #1105 I guess user missed the "If you're comfortable getting up and running from a `git clone`" part, and probably the readme file isn't clear enough and should be removed from the archive or an additional "README_to_install_on_linux" file should be provided.

- #1260 user want to contribute, I tell him to do it, I thought it was enough. :/
If he submit PR, someone will review it and should comment if things need improvements or merge it, if everything is fine.

>So please do not underestimate the importance of documentation in a project.
I don't think people intentionally underestimate it but there isn't enough devs or contributors to do everything, so choices are made.

You want linux install instructions ? Here you go: https://www.reddit.com/r/PopCornTime/comments/fnw6zz/im_trying_to_install_popcorntime_from_the_app/flcnsjw/

Everyone who want to contribute is welcome... If you guys want to improve the documentation, fork the project and send PR (I tell you the same as I said in #1260). 
Use the instructions above as example to write something more easy to understand... and eventually more generic (not just for Ubuntu)

**Comment by snowc0de** [comment]
@Persei08 Sorry to distribute. I know there are not specific to PT. Can we add a guide for Linux on the website or on the README to explain how to install. I just want to tell me if that fits what you want the README to be, I'll probably try to make a PR.
Yes, sorry I shouldn't create a new issue again :grimacing:

**Comment by snowc0de** [comment]
I am closing this issue and I'll probably make a pull request in the future but I first need to succeed to install it correctly :sweat_smile:

**Comment by Persei08** [comment]
@chopin2712 , It doesn't bother me if you leave this issue open but I think I will just change the title to something more specific... related to linux installation documentation.

If you want to send a pull request then "Make yourself at home" like @team-pct often say.

**Comment by louisdoe** [comment]
Come on Persei08 !! we just need a clean way to install popcorn, and the README is a good place, nah ? You seem angry and your response is not helping.
We are confused, because the text is confusing in the README you know.

Thanks for the time you spent responding anyway.

> You want linux install instructions ? Here you go: https://www.reddit.com/r/PopCornTime/comments/fnw6zz/im_trying_to_install_popcorntime_from_the_app/flcnsjw/

So we need to go to another website to get the install instructions ? :) Good practices !!

Here's the working process for those who want to see it here:

Install libs:
`sudo apt update && sudo apt install unzip libcanberra-gtk-module libgconf-2-4 libatomic1
`
Create popcorn-time folder in /opt/ :
`sudo mkdir /opt/popcorn-time
`
Download PT :
`wget https://get.popcorntime.app/repo/build/Popcorn-Time-0.4.4-linux64.zip
`
Extract the zip in /opt/popcorn-time :
`sudo unzip Popcorn-Time-0.4.4-linux64.zip -d /opt/popcorn-time
`
Create symlink of Popcorn-Time in /usr/bin :
`sudo ln -sf /opt/popcorn-time/Popcorn-Time /usr/bin/popcorn-time
`
Create .desktop file (so the launcher) :
`sudo nano /usr/share/applications/popcorntime.desktop
`
paste this:
```
[Desktop Entry]  
Version = 1.0
Type = Application
Terminal = false
Name = Popcorn Time
Exec = /usr/bin/popcorn-time
Icon = /opt/popcorn-time/src/app/images/icon.png
Categories = Application;
```

@chopin2712 are you succeeding with this ? Can you PR ?

@Persei08 thanks for the link to the deb files https://ci.popcorntime.app/view/All/job/Popcorn-Time-Desktop/lastStableBuild/. That's nice to know. It should be on the main website though.

Thanks for PT, because it's a great app.

**Comment by Persei08** [comment]
@louisdoe, man, I don't think i was too rude or impolite, but sorry if I sounded like. 
I provided you the install instructions which are on reddit and then you're trolling with your "Good pratices !!".

Complaining about things is more easy than fixing them. Give us some of your time and improve the things you think should be improved.  
It's an open-source project, so, if you want a thing done well, do it yourself... and have in mind there isn't enough contributors to do all the things the users are asking.

Now you have the choice, pointing at what the other have done wrong and troll about it, or helping to solve issues.
